backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/appsearchmodal/BpkAppSearchModal.kt

```diff
@@ -58,6 +58,7 @@ data class BpkItem(
     val subtitle: AnnotatedString,
     val icon: BpkIcon,
     val onItemSelected: () -> Unit,
+    val type: String,
```

Comment:
let's use `tertiaryLabel` to match web/figma API table. also let's make it nullable and give it a null default, so we don't have to set these everywhere and make it a breaking API change

Reply:
please also add a screenshot test for an example with a `tertiaryLabel`